pina/model/avno.py

```diff
@@ -0,0 +1,76 @@
+"""Module Averaging Neural Operator."""
+
+from torch import nn, concatenate
+from . import FeedForward
+from .layers import AVNOLayer
+
+
+
+
+class AVNO(nn.Module):
+    """
+    The PINA implementation of the inner layer of the Averaging Neural Operator.
+
+    :param int input_features: The number of input components of the model.
+    :param int output_features: The number of output components of the model.
+    :param ndarray points: the points in the training set.
```

Comment:
what is points?

Reply:
The coordinates of the point in which the function is evaluated. It is necessary as this neural operator works in non uniform grids.